python/cudf_polars/cudf_polars/dsl/expressions/base.py

```diff
@@ -166,7 +162,7 @@ def __init__(self, dtype: plc.DataType, error: str) -> None:
         self.dtype = dtype
         self.error = error
         self.children = ()
-        self.is_pointwise = True
+        self.is_pointwise = False
```

Comment:
Makes sense that this isn't point-wise, but still curious if there is a debugging story behind this (or if you just noticed it didn't "look right")